test/aggregators/stacks-address.test.ts

```diff
@@ -1,13 +1,15 @@
 import '../setup';
 import BN from 'bn.js';
 import StacksAddress from '../../lib/aggregators/stacks-address';
-import { getUnlockedSupply } from '../../lib/core-db-pg/queries';
+import TotalSupplyAggregator, { TotalSupplyResult } from '../../lib/aggregators/total-supply';
 
 
 test('get total supply', async () => {
-  const { unlockedSupply, blockHeight } = await getUnlockedSupply();
-  expect(unlockedSupply.gt(1)).toBeTruthy();
-  expect(parseInt(blockHeight, 10)).toBeGreaterThan(1);
+  const totalSupplyInfo: TotalSupplyResult = await TotalSupplyAggregator.fetch();
```

Comment:
We also only want to use the `setter` function - otherwise the test could return a cached value. I'll change this